packages/core/jest.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```

Comment:
Is the comment needed? 🤔

Reply:
autogenerated by jest, not sure what it does :shrug: